src/target.cpp

```diff
@@ -463,6 +464,8 @@ ZigLLVM_EnvironmentType target_abi_enum(size_t index) {
 const char *target_abi_name(ZigLLVM_EnvironmentType abi) {
     if (abi == ZigLLVM_UnknownEnvironment)
         return "none";
+    if (abi == ZigLLVM_WASISysroot)
+        return "sysroot";
```

Comment:
Not sure what to do about naming. Sysroot is super generic, but putting wasi-sysroot everywhere is super clunky `wasm32-wasi-wasisysroot`